components/matchBox/TeamNames.tsx

```diff
@@ -6,9 +6,16 @@ export type Teams = {
   away: string;
   handlePick: any;
```

Comment:
did you hover on your handle pick to see your type?

Reply:
I just removed the type declaration and hovered over handlepick - it displays "any" as type.